### DISCUSSION TITLE
Logo should stay on the navbar irrespective of screen resize

### DISCUSSION THREAD

**Discussion by siva19susi** [squidfunk/mkdocs-material#3212] [Ideas]
Currently when the page resizes, the logo moves into the Side Menu. But it makes absolute sense if it stays always on the navbar itself. Reason being, for any webpage, branding is important and moving the logo within a hidden side menu does not add any value. Moreover placing the logo always on the navbar gives the end user an upfront representation on which site one is on. 

References: microsoft.com, store.google.com, linkedin.com, apple.com

**Comment by squidfunk** [comment]
Duplicate of #1676.

**Reply by siva19susi** [reply]
Thanks for the reference.
